.prow.yaml

```diff
@@ -209,6 +209,43 @@ postsubmits:
             items:
               - key: .dockerconfigjson
                 path: config.json
+  - name: build-push-image-releaser
+    cluster: default
+    run_if_changed: 'images/releaser/.*'
```

Comment:
Is this folder will have Docker files or something? I'm trying to understand the workflow

Reply:
That's just a build for this particular image, it will contain tools like grype and trivy for image security scans, syft, crane and oras for SBOM generation and artifacts manipulation. This image will then be a part of image building 'pipeline' in general, so after kaniko finishes it will do security scans, create and upload SBOM.